app/src/main/java/org/greenstand/android/TreeTracker/viewmodels/NewTreeViewModel.kt

```diff
@@ -38,6 +38,9 @@ class NewTreeViewModel(
 
     val isTreeHeightEnabled = FeatureFlags.TREE_HEIGHT_FEATURE_ENABLED
 
+    var isNextButtonActive = !isDbhEnabled
```

Comment:
What is this boolean field for? I couldn't see its usage.

Reply:
I forgot to push my last changes. It should compile now